golang/araalictl/araalictl.go

```diff
@@ -1,4 +1,4 @@
-package main
+package araalictl
```

Comment:
will this break the main.go in this folder? Try running and testing 

go run .

Reply:
I tested the main.go in the parent directory, was working fine